test/e2e/cypress/support/commands.js

```diff
@@ -108,48 +120,37 @@ Cypress.Commands.add(
     query = {},
     fields = {}
   } = {}) => {
+    const urlRedirects = [];
+
+    cy.on('url:changed', (url) => {
+      urlRedirects.push(url);
+    });
+
     console.log('Creating user account: ', { email, password })
 
     // see https://github.com/cypress-io/cypress/issues/408
     cy.visit(APP_URL)
     cy.clearCookies()
 
-    cy.request({
-      url: APP_URL + '/self-service/registration/browser',
-      followRedirect: false,
-      qs: query
-    })
-      .then(({ redirectedToUrl }) => {
-        expect(redirectedToUrl).to.contain(APP_URL + '/auth/registration?flow=')
-        const flow = redirectedToUrl.replace(
-          APP_URL + '/auth/registration?flow=',
-          ''
-        )
-        return cy.request(
-          APP_URL + '/self-service/registration/flows?id=' + flow
-        )
-      })
-      .then(({ body, status }) => {
-        expect(status).to.eq(200)
-        const form = body.ui
-        return cy.request({
-          method: form.method,
-          body: mergeFields(form, {
-            ...fields,
-            'traits.email': email,
-            password,
-            method: 'password'
-          }),
-          url: form.action,
-          followRedirect: false
-        })
-      })
-      .then((res) => {
-        console.log('Registration sequence completed: ', { email, password })
-        expect(res.redirectedToUrl).to.not.contain(
-          APP_URL + '/auth/registration?flow='
-        )
+    cy.get('a[href*="/auth/registration"]').click()
```

Comment:
Changing the logic here will break all other tests. Please revert any changes to this method! I don't believe you need any of those here.